datadog/fakeintake/ecsFargate.go

```diff
@@ -0,0 +1,89 @@
+package fakeintake
+
+import (
+	"fmt"
+
+	"github.com/DataDog/test-infra-definitions/aws"
+	tidECS "github.com/DataDog/test-infra-definitions/aws/ecs"
+	"github.com/cenkalti/backoff/v4"
+
+	"github.com/pulumi/pulumi-awsx/sdk/go/awsx/awsx"
+	"github.com/pulumi/pulumi-awsx/sdk/go/awsx/ecs"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+const (
+	containerName = "fakeintake"
+	port          = 80
+	oneSecond     = 1000
+	sleepInterval = 1 * oneSecond
+	maxRetries    = 120
+)
+
+func ECSFargateService(e aws.Environment, name string, clusterArn string) (ipAddress pulumi.StringOutput, err error) {
+	taskDef, err := fargateLinuxTaskDefinition(e, name)
+	if err != nil {
+		return pulumi.StringOutput{}, err
+	}
+	fargateService, err := tidECS.FargateService(e, e.Namer.ResourceName(name), pulumi.String(clusterArn), taskDef.TaskDefinition.Arn())
+
+	// Hack passing taskDef.TaskDefinition.Arn() to execute apply function
+	// when taskDef has an ARN, thus it is defined on AWS side
+	ipAddress = pulumi.All(taskDef.TaskDefinition.Arn(), fargateService.Service.Name()).ApplyT(func(args []any) (string, error) {
+		var ipAddress string
+		err := backoff.Retry(func() error {
+			fmt.Println("waiting for task private ip")
```

Comment:
I think this is not displayed when calling e2e automation API

Reply:
Moving to pulumi.logs.